docs/nifgen/functions.rst

```diff
@@ -474,35 +474,23 @@ nifgen.Session methods
 
     :type coefficients_array: list of float
 
-.. py:method:: configure_digital_edge_script_trigger(trigger_id, source, edge=nifgen.ScriptTriggerDigitalEdgeEdge.RISING)
+.. py:method:: configure_digital_edge_script_trigger(source, edge=nifgen.ScriptTriggerDigitalEdgeEdge.RISING)
 
     Configures the specified Script Trigger for digital edge triggering.
 
     
 
 
+    .. tip:: This method requires repeated capabilities (usually channels). If called directly on the
```

Comment:
We will need to either add or infer more information than we do now. At this point, all we know is that there is a repeated capability used in the function, not which one. It is up to the use to know whether to use `.channel[]` or `.script_trigger[]`.

Because we don't have this additional information (yet?) we have no way to specialize the tip.